modules/orchestrator/proc-troubleshooting-missing-workflows.adoc

```diff
@@ -0,0 +1,101 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-troubleshooting-missing-workflows_{context}"]
+= Troubleshooting workflows missing from the {product-very-short} UI
+
+You can perform the following checks to verify the workflow status and connectivity when the deployed workflow is missing from the {product-very-short} Orchestrator UI.
+
+.Prerequisites
+
+* You have administrator privileges to access to the OpenShift cluster where {product-very-short} and SonataFlow services are running.
+
+.Procedure
+
+. Check if the workflow uses GitOps profile. The {product-very-short} Orchestrator UI displays only the workflows that use GitOps profile. Make sure the workflow definition and the sonataflow manifests use this profile.
+
+. Verify that the workflow pod has started and is ready. The readiness of a workflow pod depends on its successful registration with the Data Index. When a workflow initializes, it performs the following actions:
+.. It attempts to create its schema in the database (if persistence is active).
+.. It attempts to register itself to the Data Index. The workflow pod remains in an unready state until it successfully registers to the Data Index.
```

Comment:
@gabriel-farache what do you think about adding log messages indicating above actions?